specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -129,16 +176,25 @@ Example use case:
 
 ![example-use-case](example.png)
 
+### Availability
+The availability of the extension will be managed by JupyterLab since it is going to be run as part of the JupyterLab ecosystem.
 
+### Test Plan
+* The front-end extension is using Jest for JavaScript code testing.
+* This extension uses Playwright for the integration tests (aka user level tests). More precisely, the JupyterLab helper Galata is used to handle testing the extension in JupyterLab.
+* The server extension is using Pytest for Python code testing.
+* The VDK notebook plugin will be using Pytest for Python code testing.
+
+### Security and Permissions
+In terms of security, Jupyter uses tornado to ensure only authorized user can request the Jupyter server
+You can read more [here](https://jupyterlab-server.readthedocs.io/en/stable/api/index.html).
```

Comment:
Making sure to supply references is very important. But in this case I think https://jupyter-notebook.readthedocs.io/en/stable/security.html# is more precise reference for this section.